etna/transforms/timestamp/holiday.py

```diff
@@ -83,27 +146,47 @@ def _transform(self, df: pd.DataFrame) -> pd.DataFrame:
         -------
         :
             pd.DataFrame with added holidays
+
+        Raises
+        ------
+        ValueError:
+            if the frequency is greater than daily and this is a ``binary`` or ``categorical`` mode
+        ValueError:
+            if the frequency is not weekly, monthly, quarterly or yearly data and this is ``days_count`` mode
```

Comment:
It seems like this expression is incorrect. It talks about both frequency and data.